docs/articles/sdk/configure-qtcreator.rst

```diff
@@ -123,6 +123,12 @@ Kit
   the result of the ``echo ${SDKTARGETSYSROOT}`` command ran in an SDK terminal
 * Choose the previously defined PELUX C and C++ compilers from the corresponding combo boxes
 * Change the environment of this kit to the result of the ``env`` command ran in an SDK terminal
+    * If ``CC`` and ``CXX`` have default compiler flags, those should be added in ``CFLAGS`` and ``CXXFLAGS``
+      To replace ``CFLAGS`` and ``CXXFLAGS``, use the result of the below command ran in
```

Comment:
s/ran/run/

Reply:
@thenor Thanks for the review. 
* `s/command/commands/` This seems to be fixed.
* `s/ran/run` <- Should I fix it? Because the above other similar sentences also use a same tense. If I'm wrong, please let me know. I'm ok to change this as well.